admin/customers.php

```diff
@@ -1370,8 +1370,9 @@ function init() {
                   $contents[] = array('align' => 'text-center', 'text' => '<br><button type="submit" class="btn btn-warning">' . IMAGE_RESET_PWD . '</button> <a href="' . zen_href_link(FILENAME_CUSTOMERS, zen_get_all_get_params(array('cID', 'action')) . 'cID=' . $cInfo->customers_id) . '" class="btn btn-default" role="button">' . IMAGE_CANCEL . '</a>');
                   break;
                 default:
-                  if (isset($_GET['search']))
+                  if (isset($_GET['search'])) {
                     $_GET['search'] = zen_output_string_protected($_GET['search']);
+                  }
```

Comment:
one liner maybe?

Reply:
FWIW, except for the case of checking for undefined constants, my preference for processing conditionals is to use the braces to help identify the full clause.  I can't count the number of times where I've missed one of those one-liners when trying to reformat a module.